extensions/wikia/Recirculation/js/discussions.js

```diff
@@ -0,0 +1,57 @@
+/*global require*/
+require([
+	'jquery',
+	'wikia.abTest',
+	'wikia.tracker',
+	'wikia.nirvana',
+], function ($, abTest, tracker, nirvana) {
+	console.log('DISCUSSIONS');
+
+	function trackClick(location) {
+		tracker.track({
+			action: tracker.ACTIONS.CLICK,
+			category: 'recirculation',
+			label: 'discussions-' + location,
+			trackingMethod: 'analytics'
+		});
+	}
+
+	function trackImpression() {
+		tracker.track({
+			action: tracker.ACTIONS.IMPRESSION,
+			category: 'recirculation',
+			label: 'discussions',
+			trackingMethod: 'analytics'
+		});
+	}
+
+	function injectDiscussions(cb) {
```

Comment:
`done`, `onDone`, `callback` would be better names I think. `cb` means cache buster to me